modules/core/src/passes/layers-pass.ts

```diff
@@ -206,7 +206,7 @@ export default class LayersPass extends Pass {
         moduleParameters.viewport = viewport;
 
         try {
-          layer.drawLayer({
+          layer._drawLayer({
```

Comment:
Nit: Is renaming methods at this stage (lots of big PRs, minor release) work the risk of breakages?